cli/Fixtures/issue6096/README.md

```diff
@@ -0,0 +1,6 @@
+# Framework with native Swift Macro and Plugin packages
```

Comment:
I'd rename the directory from `issue_6096` to something more representative and keep it scoped to just plugins. Perhaps `generated_mac_app_with_native_plugins`.